apps/frontend/src/app/5_pages/EarnPage/EarnPage.tsx

```diff
@@ -438,6 +441,24 @@ const EarnPage: FC = () => {
             />
           )}
 
+          <SimpleTable className="mt-3">
+            <SimpleTableRow
+              label={
+                <span className="relative">
+                  {t(pageTranslations.subsidiesRewardRate)}
+                  <HelperButton
+                    className="absolute -right-1 top-0 translate-x-full"
+                    content={t(pageTranslations.subsidiesRewardRateInfo)}
+                  />
+                </span>
+              }
+              valueClassName={classNames('transition-colors', {
+                'text-primary-10': !isAmountZero,
+              })}
+              value={`${formatValue(apy, 5)}% APY`}
```

Comment:
I already mentioned it in the story but the value should be rounded to 2 decimals.